### DISCUSSION TITLE
Download everything at once without having to confirm one by one

### DISCUSSION THREAD

**Discussion by rciptv2019** [UltimaHoarder/UltimaScraper#1254] [Q&A]
Would it be possible to download everything from all accounts without having to confirm one by one?

**Comment by UltimaHoarder** [comment]
Yes, read the readme. Specifically the auto choices.

**Reply by rciptv2019** [reply]
Thank you. I look at it now to see how it is.

**Reply by Kroganil** [reply]
You need to set auto_media_choice:  to 0
`"auto_media_choice": "0",`
